pkg/images/images_test.go

```diff
@@ -236,7 +236,7 @@ func Test_releaseToKeep(t *testing.T) {
 			name: "min = 2.7.0 < max = 2.9.0",
```

Comment:
need to be updated to 
```suggestion
			name: "min = 2.7.0 < max = 2.10.0",
```